content/6_Advanced/SCC.mdx

```diff
@@ -202,11 +202,88 @@ int main() {
 
 <FocusProblem problem="satEx" />
 
-<IncompleteSection>
+Model each topping as a boolean variable and each wish as a clause, so all wishes define a boolean formula in 2-CNF. Then we rewrite each clause $x_i \vee x_j$ as $(\bar x_i \rarr x_j) \wedge (\bar x_j \rarr x_i)$ and construct the implication graph with each variable and it's negation as a vertex.
```

Comment:
```suggestion
Model each topping as a boolean variable and each wish as a clause, so all wishes define a boolean formula in 2-CNF. Then we rewrite each clause $x_i \vee x_j$ as $(\bar x_i \rarr x_j) \wedge (\bar x_j \rarr x_i)$ and construct the implication graph with each variable and its negation as a vertex.
```

A diagram here would be nice

Reply:
Also, add another paragraph connecting 2-SAT and SCCs, and describe which algorithm you use to solve SCC in the code provided